code-challenges.js

```diff
@@ -31,24 +35,65 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const q1 = (temp) => {
```

Comment:
Logic and pseudocode looks excellent!  I really like that you considered edge cases in your logic and checked it on line 52.  One suggestion is to name your function according to the intent of the problem not the number of the problem.  An example for this problem would be checkBoilingPoint for the function name.